uaclient/cli.py

```diff
@@ -165,36 +178,33 @@ def action_disable(args, cfg):
         return 1
 
 
+@assert_attached_root
 def action_enable(args, cfg):
     """Perform the enable action on a named entitlement.
 
     @return: 0 on success, 1 otherwise
     """
+    if cfg.is_attached and os.getuid() == 0:
```

Comment:
We no longer need this check due to `assert_attached_root`

Reply:
cgood point, dropped, we had discussion about potentially not performing the pre-enable refresh check because it costs 7 seconds
@powersj will weigh in on whether this branch should drop the pre-enable refresh check.
  - Client could handle stale auth or url errors on setup here and emit 'Try ua refresh' to the user if enable didn't work for instance.

Reply:
We won't address the separationg of pre-enable apt-get update in this branch. we will talk about it in ua status weekly checkin meeting